chain/pool/txpool.go

```diff
@@ -30,6 +30,20 @@ func NewTxPool() *TxnPool {
 }
 
 func (tp *TxnPool) AppendTxnPool(txn *transaction.Transaction) error {
+	sender, err := txn.GetProgramHashes()
+	if err != nil {
+		return err
+	}
+
+	list, err := tp.getOrNewList(sender[0])
```

Comment:
This will cause panic if someone sent a txn without signature

Reply:
sender, err := txn.GetProgramHashes(), if err != nil ,then return.